source/sass/_colors.scss

```diff
@@ -28,7 +28,8 @@ $pni-purple-pink: #b0379b;
 $pni-pink: #e4487d;
 $pni-gradient: linear-gradient(to right, $pni-blue 0%, $pni-pink 100%);
 
-$bg-product-image: #f4f4f4; /* keep this #f4f4f4 as this is the background colour that product images have */
+$pni-bg-product-list: #f5f5f5; /* keep this #f5f5f5 as this is the background colour product images on Homepage have */
```

Comment:
the `bg` is "buyersguide", so we probably want to replace `bg` with `pni`, rather than adding it at the front?

Reply:
Ah I used `bg` for `background`. Would replacing `bg` here with `background` help?

Reply:
Ah I used `bg` for `background`. I will replace `bg` here with `background` to make it more explicit.

Reply:
Updated!